### ISSUE TITLE
Error to open the StdDriver sample code

### ISSUE THREAD

**Issue by eddylai** [OpenNuvoton/M031BSP#2]
error to open example code in SampleCode\StdDriver folder with uVision 5

Error: Devcie not found - 
   Device:  'M032SE3AE'
   Vendor: 'Nuvoton'
Please update your device selection,

but not able to find the m031 device from project Options

**Comment by kchuang1** [comment]
Hi,

Do you install the Nuvoton Keil driver from below link?
https://www.nuvoton.com/tool-and-software/ide-and-compiler/

**Comment by eddylai** [comment]
> 
> 
> Hi,
> 
> Do you install the Nuvoton Keil driver from below link?
> https://www.nuvoton.com/tool-and-software/ide-and-compiler/

yes, have installed Nu-Link_IAR_Driver 3.07.7246r and can flash axf to board

**Comment by kchuang1** [comment]
Please install Nu-Link_Keil_Driver V3.07.224r6, not Nu-Link_IAR_Driver.

**Comment by eddylai** [comment]
> 
> 
> Please install Nu-Link_Keil_Driver V3.07.224r6, not Nu-Link_IAR_Driver.

oh, yes , I have installed the  Nu-Link_Keil_Driver_V3.07.7246r driver, not IAR
found maybe it's because I use MDK-ARM 5.35.0.2 , what version do you use?

**Comment by kchuang1** [comment]
My environment is MDK-ARM Plus V5.24 or V5.28. They all are OK.
